README.md

```diff
@@ -5,40 +5,37 @@
 - **Field Name Prefix:** storage
 - **Scope:** Item, Asset
 - **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):** Proposal
-- **Owner**: @davidraleigh
+- **Owner**: @davidraleigh @matthewhanson
 
 This document explains the Storage Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
 This extension adds fields to STAC Item and Asset objects, allowing for details related to cloud storage access and costs to be associated
 with a STAC Item.  This extension does not cover NFS solutions provided by PaaS cloud companies.
 
 - Examples:
-  - [Item example 1](examples/example-naip.json): Shows the basic usage of the extension in a STAC Item.
-  - [Item example 2](examples/example-nsl.json): Another example of basic usage.
+  - [Item example 1](examples/item-naip.json): Shows the basic usage of the extension in a STAC Item.
+  - [Item example 2](examples/item-nsl.json): Another example of basic usage.
 - [JSON Schema](json-schema/schema.json)
 - [Changelog](./CHANGELOG.md)
 
 ## Item Fields
+
 | Field Name  | Type   | Description |
 | ----------- | ------ | ----------- |
 | storage:min_tier_duration   | integer  | number of days for the shortest time tier restriction on access of an asset. |
 | storage:max_tier_duration   | integer  | number of days for the longest time tier restrictions on access of an asset. |
 | storage:archived            | bool    | descriptor for whether the data is "properly" archived according to implementers discretion |
-
-## Asset Fields
-
-| Field Name  | Type   | Description |
-| ----------- | ------ | ----------- |
 | storage:platform              | string    | (REQUIRED) The [cloud provider](#providers) where data is stored |
```

Comment:
`platform` at Item level won't work if you have multiple assets in different cloud platforms

Reply:
As per above comment, you can specify `storage:platform` at the Item level and/or the asset level